prowler/providers/aws/services/elasticache/elasticache_redis_cluster_below_v6_auth_enabled/elasticache_redis_cluster_below_v6_auth_enabled.metadata.json

```diff
@@ -0,0 +1,32 @@
+{
+  "Provider": "aws",
+  "CheckID": "elasticache_redis_cluster_below_v6_auth_enabled",
+  "CheckTitle": "Ensure Elasticache Elasticache Redis replication groups under version 6.0 have Redis OSS AUTH Enabled.",
+  "CheckType": [
+    "AWS Foundational Security Best Practices v1.0.0"
+  ],
```

Comment:
Please, use the types from https://docs.aws.amazon.com/securityhub/latest/userguide/asff-required-attributes.html#Types